infrastructure/scripts/metadata-migration/create-metadata-app-version.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -eu # protection
+#set -x # enable debug logs
+
+echo --------------------Versions------------------------------
+for env in environments/* # $(find environments  -maxdepth 1 -mindepth 1 -type d)
+do
+  for app in "$env"/applications/*
+  do
+    # If $app has version
+    echo Application Version: "$app"
+    if [ -d "$app"/version ]; then
+      git log -1 --date=iso-strict --format="%ad" -- "$app"/version > "$app"/deploy_date
```

Comment:
I recall that we discussed removing this date entirely. Do we still need it?

Reply:
I thought we could use it in the future as the deploy date. and as this is a one-time migration, creating this now would be better? 

also, the backend can write the deploy date for newer deployments before we find a good way to show it in the UI. 

I think the feature is nice but it's not urgent. 
in the future:
* we can uncomment this field in the API and read the date from the repository. 
* then it's available in the frontend. 
* then we find a good way to show this in the UI.